tools/sentencepiece_testing/create_bad_proto.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 The KerasNLP Authors
```

Comment:
maybe rename this file `create_bad_proto.py` -> `create_no_special_token_proto.py`

Reply:
done, thanks!